core/platform/search/elastic_search_services.py

```diff
@@ -23,17 +23,16 @@
 
 import json
 
+import elasticsearch
 import feconf
 import python_utils
 
-import elasticsearch
-
 # https://www.elastic.co/guide/en/elasticsearch/reference/current/index-modules.html#index-max-result-window
-# This is the maximum number of pages to page through using the from and size
-# parameters to page through any given index. This number is equal to the size
-# index.max_result_window and defines the maximum the sum of the parameters
-# (size + from) can be. If needed, we will have to change this to use the
-# search_after parameter or scrolling option as mentioned in the link above.
+# This is the maximum number of results to search through using the from and
```

Comment:
Put quotes around "from" and "size".

Also, the wording is still a bit confusing. Is this the max number of docs to search through or the maximum number of results that will be returned for a search query? I don't see how the search algorithm has any control over the former, so presumably you mean the latter.

Reply:
Yep, the search algorithm gives the maximum number of 'hits' or results.

Reply:
Fixed the doc. It now reads: "This is the maximum number of results that can be returned for any given search query."